src/main/java/com/faforever/client/chat/ChatUserControl.java

```diff
@@ -108,6 +245,25 @@ private void configureAvatarImageView() {
     Tooltip.install(avatarImageView, avatarTooltip);
   }
 
+  private void configureGameStatusView() {
+    setGameStatus(playerInfoBean.getGameStatus());
+    playerInfoBean.gameStatusProperty().addListener((observable, oldValue, newValue) -> {
+      Platform.runLater(() -> setGameStatus(newValue));
+    });
+  }
+
+  @FXML
+  void onMouseEnterGameStatus() {
+    if (playerInfoBean.getGameStatus() != GameStatus.NONE) {
```

Comment:
Prefer:

    if(abortCondition) {
      return;
    }
    // lots of code

over:

    if(continueCondition) {
      // lots of code
    }

since it only adds unnecessary indentation